knowledge-base/grid-filter-by-selected-items.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Filter the Grid by selected items
+description: How to filter the Grid by Selected Items?
+type: how-to
+page_title: Filter the Grid by selected items
+slug: grid-kb-filter-by-selected-items
+position: 
+tags: grid,filter,selected,only
```

Comment:
Tags are not keywords, but more like categories. They should make sense in isolation.

```suggestion
tags: grid,filter,selected
```